startstoptorq.sh

```diff
@@ -0,0 +1,275 @@
+# Load the environment
+. ./setenv.sh
+
+# sets the base port for a default TorQ installation
+export KDBHDB=${PWD}/hdb/database
+export KDBWDB=${PWD}/wdbhdb
+export KDBSTACKID="-stackid ${KDBBASEPORT}"
+export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$KDBLIB/l32
+export DEFAULTCSV=${KDBAPPCONFIG}/process.csv
+
+getfield() {
+  fieldno=`awk -F, '{if(NR==1) for(i=1;i<=NF;i++){if($i=="'$2'") print i}}' $csvpath` 		    # get number for field based on headers
+  fieldval=`awk -F, '{if(NR == '$1') print $'$fieldno'}' $csvpath`                    		    # pull one field from one line of file
+  echo $fieldval | envsubst                                                                         # substitute env vars
+ }
+
+parameter() {
+  fieldval=`getfield $1 $2`
+  if [ "" == "$fieldval" ]; then                                                                    # check for empty string
+    echo ""
+  else
+    echo " -"$2 $fieldval
+  fi
+ }
+
+getqcmd() {
+  fieldval=`getfield $1 $2`
```

Comment:
seems like this is a function for getting a specific field, why does it need to have two params? Surely second is always going to be `"qcmd"`, why not just hardcode this in here?